Spelling Checker/src/SpellingsSuggestionList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SpellingSuggestion } from './SpellingSuggestion';
+
+export const SpellingsSuggestionList = (props) => {
+    const data = props.correctSpellingList;
+    const textVal = props.text; //console.log(textVal);
+    const setText = props.setText; //console.log('text value : ',textVal);
+    return (
+        <div>
+            {
+                data.map((spellingData) => {
+                    console.log('Suggestion for wrong/similar words : ', spellingData);
+                      return (
+                          <SpellingSuggestion
+                            //key = {id}
```

Comment:
`key` is a good thing to have.

Reply:
Sure, understood. I was getting warnings too.